rest_framework_gis/fields.py

```diff
@@ -41,3 +41,8 @@ def validate_empty_values(self, data):
         if data in [u'']:
             self.fail('required')
         return super(GeometryField, self).validate_empty_values(data)
+
+class GeometrySerializerMethodField(SerializerMethodField):
+    def to_representation(self, value):
+        value = super(GeometrySerializerMethodField, self).to_representation(value)
```

Comment:
does it really need to go through `value = super(GeometrySerializerMethodField, self).to_representation(value)`.